tests/terratest/config/config.go

```diff
@@ -0,0 +1,91 @@
+package config
+
+import (
+	"github.com/rancher/rancher/tests/terratest/models"
+)
+
+// Modules
+var Aks                = "aks"
+var K3s                = "k3s"
+var Rke1               = "rke1"
+var Rke2               = "rke2"
+
+// K8s versions
+var AKSK8sVersion1235  = "1.23.5"
+var AKSK8sVersion1226  = "1.22.6"
+var AKSK8sVersion1219  = "1.21.9"
+
+var K3sK8sVersion1237  = "v1.23.7+k3s1"
```

Comment:
Are we going to need to continuously update these k8s values? Could this be added as a configuration file that dynamically loads these values from ?

Could it be the same way the golang framework loads a conf file and is used as a variable in the Jenkins execution.

Reply:
Yes, i suppose that would be fine as well.. they are fixed values, so they wouldn't need to be updated per-say, but new versions would need to be declared for future use.  These values are only being referenced within the test files to build the TF  main.tf files, so as long as the test is grabbing the right values, they could come from anywhere.  Once declared, they will always be available to use in any test, so adding new ones will be necessary, but no reason to change versions, unless deprecated, then they could be removed from this config.go file.  

Either way would work fine, i initially thought of this config.go as a "component" config library with defined, reusable values to build tests with.

Reply:
I feel like its best the way I have it, because if externally provided, i think it would hard to keep consistency between tests being written, like naming conventions and knowing which are needed to run specific tests, as it wouldn't be available for reference anywhere.  You would need to examine the test, to see what the variables are called and then figure out the appropriate values.  With these values centralized and defined in the repo's config.go, it is easy to reference both the key:value in the config, as well as the reference to that value in the tests.